packages/wrangler/src/logger.ts

```diff
@@ -65,6 +65,10 @@ export class Logger {
 		this.overrideLoggerLevel = val;
 	}
 
+	resetLoggerLevel() {
+		this.overrideLoggerLevel = undefined;
+	}
+
```

Comment:
I have no idea where I see `resetLoggerLevel` before you make this changes now... 😅